Fix/config wizard and issues

rsoury opened the pull request.

rsoury committed:
update config wizard for new schema and options

rsoury committed:
add descriptions to base config

rsoury committed:
add storage-proxy to repo

rsoury committed:
update lock

rsoury committed:
update dependency on storage-proxy to local workspace

rsoury committed:
update node readme

rsoury committed:
add @types/node to storage-proxy

rsoury committed:
update config examples, and reference to non-standalone configs in test

victorshevtsov commented on packages/core/src/config/config.schema.json:
Why logStoreClient and streamrClient sections removed from the schema?
Thses sections are required to validate corresponding configs and pass it to the clients costructors.
DevNetwork does not start with that changes.

rsoury committed:
revert config updates, and update config wizard for new schema.

rsoury committed:
format using prettier

rsoury committed:
small fix in logging

victorshevtsov committed:
Fix tests for message processing by a program

victorshevtsov committed:
Skip the config test with minimal config

The test skips because LogStore Node cannot run just with minimal config.
The node requires several steps to perform prior starting i.e. join the network.
Such required steps are covered by other tests.

victorshevtsov committed:
Improve tear doewn if a test fails

victorshevtsov committed:
Run the tests sequentially to avoid conflicts when port listening and calling smart contracts.

victorshevtsov committed:
Fix BucketManager test that prevents jest from exiting cleanly

victorshevtsov committed:
Fix LogStoreConfig test

victorshevtsov committed:
Fix failing test of http endpoint for query requests

rsoury committed:
attempt vitest over jest?

rsoury committed:
remove vitest - tests in jest, now passing more

rsoury committed:
add TODO to matter inside of storage proxy
